Documentation: add information about existing documentation and APIs (#1009)
ansys/pydpf-core: docs/source/index.rst

Each DPF capability is developed through operators that allow for componentization
of the framework. Because DPF is plugin-based, new features or formats can be easily added.

Accessing and enriching DPF capabilities
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Most of the DPF capabilities can be accessed using the operators.

> For more information about the existing operators, see **Operators** tab.

Reviewer: ```suggestion
For more information about the existing operators, see the **Operators** tab.
```

> Following sections are summarizes. For more detailed content, see :ref:`user_guide_waysofusing`.

Reviewer: ```suggestion
The following sections are summaries. For more detailed content, see :ref:`user_guide_waysofusing`.
```

**Accessing DPF Server files**

> DPF capabilities are accessible having DPF Server files available. Those files can be accessed using:

Reviewer: ```suggestion
DPF capabilities are accessible when DPF Server files are available. These files can be accessed using:
```

> - the **Ansys installer**. To use it, download the standard installation using your preferred distribution channel,

Reviewer: ```suggestion
- The **Ansys installer**. To use it, download the standard installation using your preferred distribution channel,
```

and install Ansys following the installer instructions. For information on getting a licensed copy of Ansys,
visit the `Ansys website <https://www.ansys.com/>`_.

> - the DPF Server package (see :ref:`ref_getting_started_with_dpf_server`).

Reviewer: ```suggestion
- The DPF Server package (see :ref:`ref_getting_started_with_dpf_server`).
```

It is independent of the Ansys installer.

**Accessing capabilities with scripting**

> - C++ documentations:

Reviewer: ```suggestion
- C++ documentation:
```

>   1. see Data Processing Framework section in `Platform panel <https://ansysapi.ansys.com/account/secured?returnurl=/Views/Secured/main_page.html?lang=en>`_.

Reviewer: ```suggestion
  1. The Data Processing Framework section in `Platform panel <https://ansysapi.ansys.com/account/secured?returnurl=/Views/Secured/main_page.html?lang=en>`_.
```

>   2. see `Developer Portal <https://developer.ansys.com/product/DPF-Server-Client-Library/index.xhtml>`_

Reviewer: ```suggestion
  2. `Developer Portal <https://developer.ansys.com/product/DPF-Server-Client-Library/index.xhtml>`_
```

> - CPython modules documentations:

Reviewer: ```suggestion
- CPython modules documentation:
```
